python/multicorn/sqlalchemyfdw.py

```diff
@@ -362,6 +362,8 @@ def __init__(self, fdw_options, fdw_columns):
 
         self.engine = _create_engine(url, self.connect_args)
 
+        self.cast_quals = fdw_options.get("cast_quals", "no") == "yes"
```

Comment:
Use `true`/`false` here to be in line with e.g. postgres_fdw (https://www.postgresql.org/docs/13/postgres-fdw.html#id-1.11.7.42.10)